src/components/toolbar/toolbar.js

```diff
@@ -56,12 +56,17 @@ angular.module('material.components.toolbar', [
  * at one fourth the rate at which the user scrolls down. Default 0.5.
  */
 
-function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate ) {
-  var translateY = angular.bind(null, $mdUtil.supplant, 'translate3d(0,{0}px,0)' );
+function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate) {
+  var translateY = angular.bind(null, $mdUtil.supplant, 'translate3d(0,{0}px,0)');
 
   return {
     restrict: 'E',
+    template: '<div ng-transclude></div>',
```

Comment:
Heh...actually, it looks like we can just remove the `if` altogether. The setup function already checks whether it should enable/disable, so we just have to call it.